k8s/agent-pre-2.1.1/README.md

```diff
@@ -1,9 +1,9 @@
 ## Kubernetes Files for Versions Before 2.1.1
 
-Detailed instructions for installing and upgrading the Scalyr agent in kubernetes can be found [here](https://app.scalyr.com/help/install-agent-kubernetes).
+Detailed instructions for installing and upgrading the Scalyr Agent in kubernetes can be found [here](https://app.scalyr.com/help/install-agent-kubernetes).
```

Comment:
Insert before the "Detailed instructions" line this text:

This directory contains the manifest files used for installing the Scalyr Agent prior to the 2.1 Scalyr Agent release.  You should only use these manifest files if you are running an instance prior to 2.1.

Here's a question... do we need this directory if we have the `default-namespace` one?  You were just using this as a placeholder to perform the delete... but the `default-namespace` would suffice for that as well?